client/java/src/test/java/io/openlineage/client/OpenLineageTest.java

```diff
@@ -49,26 +47,27 @@ public void jsonSerialization() throws JsonProcessingException {
     URI producer = URI.create("producer");
     OpenLineage ol = new OpenLineage(producer);
     UUID runId = UUID.randomUUID();
-    RunFacets runFacets = ol.newRunFacetsBuilder().nominalTime(ol.newNominalTimeRunFacet(now, now)).build();
+    RunFacets runFacets =
+        ol.newRunFacetsBuilder().nominalTime(ol.newNominalTimeRunFacet(now, now)).build();
     Run run = ol.newRun(runId, runFacets);
     String name = "jobName";
     String namespace = "namespace";
     JobFacets jobFacets = ol.newJobFacetsBuilder().build();
     Job job = ol.newJob(namespace, name, jobFacets);
     List<InputDataset> inputs = Arrays.asList(ol.newInputDataset("ins", "input", null, null));
     List<OutputDataset> outputs = Arrays.asList(ol.newOutputDataset("ons", "output", null, null));
-    RunEvent runStateUpdate = ol.newRunEvent(OpenLineage.RunEvent.EventType.START, now, run, job, inputs, outputs);
-
+    RunEvent runStateUpdate =
+        ol.newRunEvent(OpenLineage.RunEvent.EventType.START, now, run, job, inputs, outputs);
 
     String json = mapper.writeValueAsString(runStateUpdate);
     RunEvent read = mapper.readValue(json, RunEvent.class);
 
-    assertEquals(producer,read.getProducer());
-    assertEquals(runId,read.getRun().getRunId());
-    assertEquals(name,read.getJob().getName());
-    assertEquals(namespace,read.getJob().getNamespace());
-    assertEquals(runStateUpdate.getEventType(),read.getEventType());
-    assertEquals(runStateUpdate.getEventTime(),read.getEventTime());
+    assertEquals(producer, read.getProducer());
+    assertEquals(runId, read.getRun().getRunId());
+    assertEquals(name, read.getJob().getName());
+    assertEquals(namespace, read.getJob().getNamespace());
+    assertEquals(runStateUpdate.getEventType(), read.getEventType());
+    assertEquals(runStateUpdate.getEventTime(), read.getEventTime());
```

Comment:
Which piece of transport feature is this test trying to cover?

Reply:
This PR for some reason reformatted existing tests for OpenLineage event serialization.